src/conspiracies/docproc/coref/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .coref_model import CoreferenceModel  # noqa
+from .coref_comp import CoreferenceComponent, create_coref_component  # noqa
```

Comment:
```suggestion
from .coref_component import CoreferenceComponent, create_coref_component  # noqa
```
again would write it out (your editor autocompletes it anyway) and it increases readability